docs/schema/commanduihandlers-element-commanduiextensiontype-complextypeapphostwebfeatures.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: CommandUIHandlers element (CommandUIExtensionType complexType)
+description: Describes the CommandUIHandlers element (CommandUIExtensionType complexType), including element information, definition, attributes, and parent & child elements.
```

Comment:
Too long per https://review.docs.microsoft.com/en-us/help/contribute/contribute-how-to-write-seo-meta-description?branch=main.